### PR TITLE
Installation updates and 1.0.10

### PR DESCRIPTION
**Pull request by neal-timpe**
Fixes to the install flow and version updates.

### CODE REVIEW

**Comment by neal-timpe** [comment]
@JStickler Can you take a look when you get a chance?

File: modules/ossm-member-roll-create.adoc [hunk 1]
```diff
@@ -22,11 +22,19 @@ Follow this procedure to add one or more projects to the {ProductShortName} memb
 
 .Prerequisites
 * An installed, verified {ProductName} Operator.
-* Location of the installed `ServiceMeshControlPlane`.
-* List of projects to add to the service mesh.
```
Comment (JStickler) [inline modules/ossm-member-roll-create.adoc]: * List of existing projects that you want to have the service mesh monitor.

**Review by JStickler — CHANGES_REQUESTED** [review]
Once you make the updates from our discussion today ping me on Slack and we'll get this merged.

File: modules/ossm-member-roll-create.adoc (hunk 1 above)
Comment (neal-timpe) [reply modules/ossm-member-roll-create.adoc]: The changes you requested have been updated, @JStickler and I marked them resolved. This isn't ready to merge until 1.0.10 is released.

**Comment by JStickler** [comment]
LGTM.

**Review by JStickler — APPROVED** [review]

**Pull request merged**